cdc/pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -64,7 +64,7 @@ func (o *changefeedCommonOptions) addFlags(cmd *cobra.Command) {
 		return
 	}
 
-	cmd.PersistentFlags().BoolVar(&o.noConfirm, "no-confirm", false, "Don't ask user whether to ignore ineligible table")
+	cmd.PersistentFlags().BoolVar(&o.noConfirm, "no-confirm", false, "Don't ask user whether to continue creating changefeed when replicate lag tis too large")
```

Comment:
```suggestion
	cmd.PersistentFlags().BoolVar(&o.noConfirm, "no-confirm", false, "Don't ask user to confirm warnings")
```

Please also change the "no-confirm" argument message for `changefeed resume` (in `cli_changefeed_resume`).